astropy/io/ascii/daophot.py

```diff
@@ -90,9 +90,7 @@ def parse_col_defs(self, grouped_lines_dict):
         self.col_widths = col_widths
 
         # merge the multi-line header data into single line data
-        coldef_dict = {k: sum(v, []) for (k, v) in coldef_dict.items()}
-
-        return coldef_dict
+        return {k: list(itt.chain.from_iterable(v)) for (k, v) in coldef_dict.items()}
```

Comment:
Here and below, I find the `sum` considerably clearer...

Reply:
Just to have the other options written out
```
return {k: itt.chain(*v) for (k, v) in coldef_dict.items()}
# or
return {k: reduce(operator.iconcat, v, []) for k, v in coldef_dict.items()}
# or, equivalently,
return {k: reduce(list.__iadd__, v, []) for k, v in coldef_dict.items()}
# or really not so bad here (though need better variable names)
return {k: [d for piece in v for d in piece] for k, v in coldef_dict.items()}
```

To be honest, none are as nice as `sum` - they should just optimize it in the standard library! - but oddly I might pick `chain(*v)` - don't know why `from_iterable` just makes it less clear...

Anyway, we're probably overthinking this - large numbers are not going to happen in `sum`

Reply:
✅, using chain.

Reply:
That looks best to me, too, if `sum` stays out.